index.js

```diff
@@ -231,7 +231,7 @@ if ((os.platform() === 'win32' && require('is-administrator')()) || process.getu
                 });
             }
 
-            // merge into single var after all potential saveConfigs finished
+            // merge into single let after all potential saveConfigs finished
```

Comment:
I think it would be more appropriate to refer to the variables as `var` instead of let in a comment :)

Reply:
My bad - had to exclude the comments - done!